### PR TITLE
Make lease_dir configurable

### PR DESCRIPTION
**Pull request by pilou-**

### CODE REVIEW

File: lago/subnet_lease.py [hunk 1]
```diff
@@ -41,20 +41,14 @@
 import lockfile
 import os
 
-import constants
-import utils
+import lago.config
+import lago.utils
```
Comment (david-caro) [inline lago/subnet_lease.py]: this better be relative, using '.' to lower the risk of name collision.

**Comment by david-caro** [comment]
ci test please

**Comment by david-caro** [comment]
:+1:  yep, we have a bunch of hardcoded values that are better configurable, thanks!

File: lago/subnet_lease.py (hunk 1 above)
Comment (pilou-) [inline lago/subnet_lease.py]: Done

**Comment by pilou-** [comment]
Updated (successfully tested on RHEL 7.2 using `patch-only` test suite).

**Comment by nvgoldin** [comment]
ci test please

**Comment by pilou-** [comment]
Oups ! I forgot to delete a test file (`src/lago/.lago`) before running the test suite, that's why the previous commit was incomplete :-/
It's done now (missing part of previous fix and `lease_dir` in `etc/lago.d/lago.conf` both added).

**Comment by nvgoldin** [comment]
ci test please

**Comment by mykaul** [comment]
ci merge please

**Comment by david-caro** [comment]
ci merge please

Not sure why it did not merge it before, it seems there was some issue with the checks and github check restrictions...

**Comment by nvgoldin** [comment]
can you rebase and I'll try merging again.

**Comment by pilou-** [comment]
@nvgoldin done

**Comment by nvgoldin** [comment]
ci test please

**Comment by nvgoldin** [comment]
ci merge please

**Pull request merged**